src/app/ui/modaldialog.js

```diff
@@ -82,7 +82,7 @@ module.exports = (title, content, ok, cancel, focusSelector) => {
 function html () {
   return yo`<div id="modal-dialog" class="${css.modal}">
   <div id="modal-background" class="${css['modalBackground']}"> </div>
-  <div class="${css['modalContent']}">
+  <div class="${css['modalContent']} bg-warning text-info">
```

Comment:
THis is changing the entire background color of the modal to Orange - it's not that readable.  Also now the tab-tool tip modal is appearing at the top of the screen - as if its a normal modal. ( but this is another issue)